### ISSUE TITLE
Default OW environment is not setup correctly, missing setup INCLUDE environment variable

### ISSUE THREAD

**Issue by jmalak** [open-watcom/setup-watcom#80]
Now default doesn't work due to missing path to header files.
@madebr is there any reason why the INCLUDE environment variable is not set?
I'll fix it, but I'm asking just to be sure.

**Comment by madebr** [comment]
Doesn't `INCLUDE` depend on what platform you are targeting?

**Comment by jmalak** [comment]
Yes, but OW has default setup as target = host then you don't need specify target anyway and don't need to setup include PATH. Compilers and linker use this setup as example if you use wcl386 main.c it creates executable for host without anything specific on command line as -bt -bcl or -l. On DOS it is creating for DOS, on OS/2 for OS2 etc.

**Comment by madebr** [comment]
Makes sense: set it to a common value, and assume users will override it in their github workflow.
I suppose you will add it to the [`environment` option](https://github.com/open-watcom/setup-watcom/blob/master/action.yml#L19)?

**Comment by jmalak** [comment]
Yes, I will add it if user requests environment setup.
I have a trouble with CMake which need have access to header files for host to check compilers features and it doesn't work.

**Comment by madebr** [comment]
It needs those variables set during configuration and during build.
It is possible to add this to your CMake toolchain file, but the build will fail.
```cmake
set(CMAKE_PLATFORM_NAME "Windows")
set(CMAKE_SYSTEM_PROCESSOR "I86")

# Modify this to the root of WATCOM on your system 
set(ENV{WATCOM} "$ENV{HOME}/owatcom2")

set(ENV{PATH} "$ENV{WATCOM}/binl64:$ENV{PATH}")
set(ENV{LIBPATH} "$ENV{WATCOM}/lib386;$ENV{WATCOM}/lib386/win")
set(ENV{INCLUDE} "$ENV{WATCOM}/h;$ENV{WATCOM}/h/win")
```

**Comment by jmalak** [comment]
As soon as INCLUDE will be setup to default all should  work OK.
CMake cross-compile stuff append target specific include path to the end of compiler line (after user options, but before INCLUDE environment variable content), but don't do it if target = host. It supposes standard installation of OW, which is necessary to proper work of handling include path list.

**Comment by jmalak** [comment]
fixed by PR #81
